tracer/test/snapshots/RabbitMQTests.3_x.verified.txt

```diff
@@ -0,0 +1,2334 @@
+﻿[
+  {
```

Comment:
Not for this PR, but I wonder if we should refactor the test and sample to at least split the 4 scenarios tested in the sample. That would make the files smaller and issues easier to understand when something change I believe

Reply:
> Not for this PR, but I wonder if we should refactor the test and sample to at least split the 4 scenarios tested in the sample. That would make the files smaller and issues easier to understand when something change I believe

Maybe, though I'm not very inclined. I don't think it would give us a _lot_ of value, and realistically, things _very rarely_ change with snapshots in a way that's not instantly obvious. e.g. "a tag was added to all the spans"

In cases where they _do_ change, yeah... it'll be fun 😅 

> I mainly reviewed the first one to be 100% honest

You're better than me - I just generated them 😅 Added a snapshot image of one for what it's worth